admin/config.js

```diff
@@ -27,7 +27,17 @@ module.exports = {
   Data: {
     allowedWords: process.env.ALLOWED_WORDS || 'aaa,bcd,dcd,tfg,bxx',
     pinSubmissionMaxAttempts: process.env.PIN_SUBMISSION_MAX_ATTEMPTS || 100,
-    helplineNumber: process.env.HELPLINE_NUMBER || '0345 278 8080'
+    helplineNumber: process.env.HELPLINE_NUMBER || '0345 278 8080',
+    validationRestartCodes: [
```

Comment:
I am not sure that this is config.  What is it?

Reply:
yeah, seems like an *odd* place to have this.  why not in the DB?

Reply:
@GuyHarwood  @jon-shipley it's updated